includes/Fragments/TemplateOutput.php

```diff
@@ -42,7 +42,7 @@ final protected function assign($name, $value)
      */
     final protected function setUpSmarty()
     {
-        $this->smarty = new Smarty();
+        $this->smarty = new Smarty\Smarty();
```

Comment:
Can we include the class in a `use` statement instead of qualifying it here?

Reply:
Sure, done in #973.